ext/pdo/pdo_dbh.c

```diff
@@ -761,25 +763,26 @@ static int pdo_dbh_attribute_set(pdo_dbh_t *dbh, zend_long attr, zval *value) /*
 				PDO_HANDLE_DBH_ERR();
 				return FAILURE;
 			}
-			if (Z_TYPE_P(value) != IS_ARRAY
-				|| (item = zend_hash_index_find(Z_ARRVAL_P(value), 0)) == NULL
-				|| Z_TYPE_P(item) != IS_STRING
-				|| (pce = zend_lookup_class(Z_STR_P(item))) == NULL
-			) {
-				pdo_raise_impl_error(dbh, NULL, "HY000",
-					"PDO::ATTR_STATEMENT_CLASS requires format array(classname, array(ctor_args)); "
-					"the classname must be a string specifying an existing class"
-					);
-				PDO_HANDLE_DBH_ERR();
+			if (Z_TYPE_P(value) != IS_ARRAY) {
```

Comment:
Comments from above apply here. (We should deduplicate this code -- but let's do it separately).